Optimizing the Security of your Skywallet.md

```diff
@@ -18,31 +18,31 @@ Hence, it is essential first to set a PIN code right after you receive your Skyw
 
 Though the PIN code is a four-digit combination, it would not be easy for an attacker/hacker to try the different combinations and work it out. 
```

Comment:
Although the PIN code is only a combination of four digits, it is very unlikely for an attacker to guess a secure PIN. 

Then continue with the next paragraph.